Add dic budget notebook

xliang576 opened the pull request:
Add a notebook for closing the carbon budget under MITgcm budgets discussion.

xliang576 committed:
Add a notebook for closing carbon budget under MITgcm budgets discussion.

xliang576 committed:
Merge branch 'ODyn-StAndrews:gh-pages' into dic_budget_notebook

gmacgilchrist commented:
This looks great, @xliang576, thanks so much for putting it together!

A couple of issues that I wonder if you could take a look at...
- some of the math and codeblocks don't seem to be rendering when I open the notebook on github --- could you check this on your end?
- you mention a correction that needs to be applied when working with `r` coordinates and "nonlinear free surface", however it is not clear exactly what this correction is doing or why it is needed. It could be confusing for someone coming in without prior knowledge. I suggest you write the notebook (for the moment) to be specific to the `r*`, nonlinear free surface configuration, and perhaps just make note of the fact that under different vertical coordinates or free surface treatments, additional adjustments need to be made to the surface cell. That way someone trying to reproduce your budget closure with a different configuration may notice this issue.
- I don't the plot of the residual on its own needs to be repeated since it is clear that is ~3 orders of magnitude lower
- in the intro section, consider pointing to other literature on budget closures (e.g. Chris Piecuch's document); also point back to the wiki page (and we can adjust that page to likewise link to the notebook).

xliang576 committed:
Added a few links and some adjustments around budgets.

xliang576 committed:
fix the math blocks

xliang576 committed:
fix the table codeblock

xliang576 commented:
Thanks, Graeme. 
- I removed the $r$ part of the code and the repeated residual plot. 
- Also added some links from ECCO to the wiki page, 
- and links between the wiki and the notebook (I typed these because they don't exist yet and hope they work).  
 
Sorry for adding too many commits! I know this is not the right way but I kept forgetting things.
